tcmur_cmd_handler.c

```diff
@@ -86,7 +86,7 @@ void tcmur_cmd_complete(struct tcmu_device *dev, void *data, int rc)
 }
 
 static void tcmur_cmd_iovec_reset(struct tcmur_cmd *tcmur_cmd,
-				  size_t data_length)
+				  uint64_t data_length)
```

Comment:
size_t is 32 bits on 32bit archs, right? If so, above we will use a uint64_t, but the iov_len field on a struct iovec is size_t, so will that cause a compilation error?

Reply:
Hmm - it does not create a compile error but I think that the code needs alignment around this fact - i.e size_t should be the basic assumption rather than uint64_t